--- gitlab.rb.orig	2017-11-10 12:33:15.165043378 +0000
+++ gitlab.rb	2017-11-10 13:29:56.012690924 +0000
@@ -10,7 +10,7 @@
 ##! URL on which GitLab will be reachable.
 ##! For more details on configuring external_url see:
 ##! https://docs.gitlab.com/omnibus/settings/configuration.html#configuring-the-external-url-for-gitlab
-# external_url 'GENERATED_EXTERNAL_URL'
+ external_url 'https://gitlab.iotaa.co.uk'
 
 ## Roles for multi-instance GitLab
 ##! The default is to have no roles enabled, which results in GitLab running as an all-in-one instance.
@@ -818,6 +818,9 @@
 # web_server['gid'] = nil
 # web_server['shell'] = '/bin/false'
 # web_server['home'] = '/var/opt/gitlab/nginx'
+nginx['custom_gitlab_server_config'] = "location ^~ /.well-known {
+    root /var/www/public/letsencrypt;
+}"
 
 ################################################################################
 ## GitLab NGINX
@@ -826,7 +829,7 @@
 
 # nginx['enable'] = true
 # nginx['client_max_body_size'] = '250m'
-# nginx['redirect_http_to_https'] = false
+nginx['redirect_http_to_https'] = true
 # nginx['redirect_http_to_https_port'] = 80
 
 ##! Most root CA's are included by default
@@ -838,8 +841,8 @@
 ##! if ssl_verify_client on, verification depth in the client certificates chain
 # nginx['ssl_verify_depth'] = "1"
 
-# nginx['ssl_certificate'] = "/etc/gitlab/ssl/#{node['fqdn']}.crt"
-# nginx['ssl_certificate_key'] = "/etc/gitlab/ssl/#{node['fqdn']}.key"
+nginx['ssl_certificate'] = "/etc/letsencrypt/live/gitlab.iotaa.co.uk/fullchain.pem"
+nginx['ssl_certificate_key'] = "/etc/letsencrypt/live/gitlab.iotaa.co.uk/privkey.pem"
 # nginx['ssl_ciphers'] = "ECDHE-RSA-AES256-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256"
 # nginx['ssl_prefer_server_ciphers'] = "on"
 
@@ -1105,7 +1108,7 @@
 # mattermost['service_tls_key_file'] = nil
 # mattermost['service_use_lets_encrypt'] = false
 # mattermost['service_lets_encrypt_cert_cache_file'] = "./config/letsencrypt.cache"
-# mattermost['service_forward_80_to_443'] = false
+ mattermost['service_forward_80_to_443'] = true
 # mattermost['service_read_timeout'] = 300
 # mattermost['service_write_timeout'] = 300
 # mattermost['service_time_between_user_typing_updates_milliseconds'] = 5000
